### PR TITLE
A: (nsfw) http://www.sexybabegirls.com/ftv-girls/jenna-sexy-and-matur…

### PR DESCRIPTION
**Pull request by Yuki2718**
…e/ (fixes https://github.com/AdguardTeam/AdguardFilters/issues/83643)
https://github.com/AdguardTeam/AdguardFilters/issues/83643

### CODE REVIEW

File: easylist_adult/adult_thirdparty.txt [hunk 1]
```diff
@@ -246,6 +246,7 @@
 ||widget.faphouse.com^$third-party
 ||widgets.comcontent.net^
 ||wp-script.com/img/banners/
+||x666.io^$third-party
```
Comment (spirillen) [inline easylist_adult/adult_thirdparty.txt:249]: You should move this one to at least https://github.com/easylist/easylist/blob/master/easylist_adult/adult_adservers.txt even I actually  personally would prefer the https://github.com/easylist/easylist/blob/master/easylist/easylist_adservers.txt

Own notes about this hosts: https://mypdns.org/my-privacy-dns/matrix/-/issues/1926

**Review by spirillen — CHANGES_REQUESTED** [review]
+ a rebase :smiley:

File: easylist_adult/adult_thirdparty.txt (hunk 1 above)
Comment (Yuki2718) [reply easylist_adult/adult_thirdparty.txt:249]: I no more see the request and an element has changed.

Comment (spirillen) [reply easylist_adult/adult_thirdparty.txt]: > I no more see the request and an element has changed.

Ok, I've just visited the site a couple of hours ago, and there the x666 was requested :wink: :confused:

**Pull request closed without merging**